python/ray/state.py

```diff
@@ -1185,19 +1185,22 @@ def available_resources():
     return state.available_resources()
 
 
-def errors(include_cluster_errors=True):
+def errors(all_jobs=True):
```

Comment:
why default this to True?

Reply:
this can cause odd behaviors

Reply:
That's the current default behavior, do you think it's something we should change?

Reply:
Updated the default behavior to `all_jobs=False` as per our offline discussion.